src/python/tempusloader/provider/sql/multinet.sql

```diff
@@ -195,6 +195,10 @@ ALTER TABLE tempus.poi ADD CONSTRAINT poi_road_section_id_fkey
 ALTER TABLE tempus.pt_stop ADD CONSTRAINT pt_stop_road_section_id_fkey
 	FOREIGN KEY (road_section_id) REFERENCES tempus.road_section;
 
+CREATE INDEX idx_road_node_geom ON tempus.road_node USING gist (geom);
+ALTER TABLE tempus.road_node CLUSTER ON idx_road_node_geom;
+CREATE INDEX idx_road_section_geom ON tempus.road_section USING gist (geom);
+ALTER TABLE tempus.road_section CLUSTER ON idx_road_section_geom;
 
```

Comment:
Thanks for the indexes.
However the cluster won't be of help here. You cannot cluster directly on a geometry, except by using functions like st_geohash